test/utils/answersHelpers/consignment/houseConsignment/ConsignmentItemAnswersHelperSpec.scala

```diff
@@ -127,27 +129,31 @@ class ConsignmentItemAnswersHelperSpec extends AnswersHelperSpecBase {
               val answers = emptyUserAnswers.setValue(page, value)
 
               val helper = new ConsignmentItemAnswersHelper(answers, hcIndex, itemIndex)
-              val result = helper.grossWeightRow.value
+              val result = helper.grossWeightRow
 
               result.key.value mustBe "Gross weight"
               result.value.value mustBe s"${value}kg"
-              val action = result.actions.value.items.head
-              action.content.value mustBe "Change"
-              action.visuallyHiddenText.value mustBe "gross weight of item 1"
-              action.href mustBe "#"
+              val action1 = result.actions.value.items.head
+              action1.content.value mustBe "Change"
+              action1.visuallyHiddenText.value mustBe "gross weight of item 1"
+              action1.href mustBe "#"
+              val action2 = result.actions.value.items(1)
+              action2.content.value mustBe "Remove"
+              action2.visuallyHiddenText.value mustBe "gross weight of item 1"
+              action2.href mustBe "#"
           }
         }
       }
     }
 
     "netWeightRow" - {
       val page = NetWeightPage(hcIndex, itemIndex)
-      "must return None" - {
-        s"when $page undefined" in {
-          val helper = new ConsignmentItemAnswersHelper(emptyUserAnswers, hcIndex, itemIndex)
-          helper.netWeightRow mustBe None
-        }
-      }
+//      "must return None" - {
```

Comment:
Can these be uncommented? Looks to be a couple more instances

Reply:
yeah, all complete now